openedx_events/learning/data.py

```diff
@@ -470,9 +470,21 @@ class ORASubmissionData:
     Attributes defined to represent event when a user submits an ORA assignment.
 
     Arguments:
-        id (str): identifier of the ORA submission.
-        file_downloads (List[ORAFileDownloadsData]): list of related files in the ORA submission.
+        uuid (str): The UUID of the ORA submission.
+        anonymous_user_id (str): Anonymous user ID of the user who submitted the ORA submission.
+        location (str): Location of the ORA submission.
+        attempt_number (int): Attempt number of the ORA submission.
+        created_at (datetime): Date and time when the ORA submission was created.
+        submitted_at (datetime): Date and time when the ORA submission was submitted.
+        answer (dict): Answer submitted by the user.
+        file_downloads (List[ORAFileDownloadsData]): List of related files in the ORA submission.
     """
 
-    id = attr.ib(type=str)
+    uuid = attr.ib(type=str)
+    anonymous_user_id = attr.ib(type=str)
+    location = attr.ib(type=str)
+    attempt_number = attr.ib(type=int)
+    created_at = attr.ib(type=datetime)
+    submitted_at = attr.ib(type=datetime)
+    answer = attr.ib(type=dict)
```

Comment:
A few comments:
1. I have enough context to know we didn't use this event before, so we can update the data without much trouble. Could we add that to the PR cover letter?
2. Should all this fields be required?

Reply:
> Should all this fields be required?

I would think that the `location` and `anonymous_user_id` fields could be optional. The `file_downloads` field is already optional by having a factory.
The other fields being part of the submission data I think they should be required, what do you think?

Reply:
> I would think that the location and anonymous_user_id fields could be optional. 
Is there a scenario in which this will not be sent by the event, or where those fields are none/empty? That's a good way of knowing if they should be optional. I agree the rest of them should be required, although, is the answer always present?

Reply:
> Is there a scenario in which this will not be sent by the event, or where those fields are none/empty?

In the submission these fields are not found, so they may not always be available.

> is the answer always present?

Yes, in a submission there will always be a dictionary with the user's response, either with the response text, or information from the attached files.

Reply:
> In the submission these fields are not found, so they may not always be available.
But they should always be available when sending the event, correct? So they are always non-empty.

> Yes, in a submission there will always be a dictionary with the user's response, either with the response text, or information from the attached files.
So what's the difference between answer and https://github.com/openedx/openedx-events/pull/341/files#diff-5cf3a0b26961626ce777acf38a3cd803cb0ab982f8f35a35d078cb9c62438fa3R450-R464?

Reply:
If answer and FileDownloads overlap, should we create a class for answer and include the FileDownloads there? So we don't send two fields with the same information. Correct me if I get it wrong.

Reply:
I followed your suggestion and updated the data classes, could you check it?